### ISSUE TITLE
Cannot Install Plugin for iOS after IOS_FIREBASE_MESSAGING_VERSION Update

### ISSUE THREAD

**Issue by PoeticArcher** [andrehtissot/cordova-plugin-fcm-with-dependecy-updated#196]
I have an automated deployment for our apps and it broke after you added the IOS_FIREBASE_MESSAGING_VERSION variable. Something isn't getting set automatically in ios for those of us with existing config.xml and package.jsons. The error is ArgumentError - Illformed requirement "$IOS_FIREBASE_MESSAGING_VERSION" after pod 'Firebase/Messaging', '$IOS_FIREBASE_MESSAGING_VERSION'

RUBY: ruby 2.6.3p62 (2019-04-16 revision 67580) [universal.x86_64-darwin19]
RUBY GEMS: 3.0.3
Xcode: 12.1
POD: 1.9.3

```
Ionic:

   Ionic CLI                     : 6.4.3 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.0.7
   @angular-devkit/build-angular : 0.803.26
   @angular-devkit/schematics    : 8.3.26
   @angular/cli                  : 8.3.26
   @ionic/angular-toolkit        : 2.2.0

Cordova:

   Cordova CLI       : 9.0.0 (cordova-lib@9.0.1)
   Cordova Platforms : android 9.0.0, ios 5.1.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 4.2.1, (and 24 other plugins)


System:

   Android SDK Tools : 26.1.1 (/Users/vmann/Library/Android/sdk/)
   ios-deploy        : 1.9.2
   ios-sim           : 8.0.2
   NodeJS            : v12.16.1 (/usr/local/bin/node)
   npm               : 6.14.5
   OS                : macOS Catalina
   Xcode             : Xcode 12.1 Build version 12A7403
```

**Comment by kelokchan** [comment]
I'm getting the same error. I tried setting IOS_FIREBASE_MESSAGING_VERSION variable when add this plugin, but the installation will fail in pod install

**Comment by sylvainPenbase** [comment]
Hi,

it's a cordova-ios bug, fixed into 6.0.0 : https://github.com/apache/cordova-ios/pull/754

**Comment by andrehtissot** [comment]
Thank you @sylvainPenbase
